store/derivatives.ts

```diff
@@ -832,12 +833,14 @@ export const actions = actionTree(
         market,
         quantity,
         price,
-        orderType
+        orderType,
+        reduceOnlyOrders
```

Comment:
Let's make a new store action `closePositionAndReduceOnlyOrders` to follow the convention we have, remove the `reduceOnlyOrders: UiDerivativeLimitOrder[]` from this store action

Reply:
new store action `closePositionAndReduceOnlyOrders` was created, and `reduceOnlyOrders: UiDerivativeLimitOrder[]` was removed from `closePosition` store action